src/Extension/CoreExtension.php

```diff
@@ -1280,6 +1280,9 @@ public static function callMacro(Template $template, string $method, array $args
 
     /**
      * @internal
+     * @template TSequence of mixed
```

Comment:
`of mixed` is useless (as `mixed` is the top type)

Reply:
You mean it's not needed?
So we can do this:
```suggestion
     * @template TSequence
```

Reply:
I like to be explicit, but fine. I will remove it 😊